UnitySDK/Assets/ML-Agents/Scripts/LearningBrain.cs

```diff
@@ -10,14 +10,14 @@ namespace MLAgents
 {
     public enum InferenceDevice
     {
-        CPU = 0,
-        GPU = 1
+        Cpu = 0,
```

Comment:
Keep the old way?

Reply:
Yeah, seems like the automatic Rider modification prefers this way, though the reference does have an enum value named `UV`.  I'll change this back.